Lombiq.Hosting.Tenants.IdleTenantManagement/Models/IdleMinutesSettings.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Lombiq.Hosting.Tenants.IdleTenantManagement.Models;
+
+public class IdleMinutesSettings
```

Comment:
This is better as an appSettings config, which is also possible to manage on a per-tenant basis. See `DemoSettings` in the Training Demo (but that also uses site settings, which are not needed here).